examples/nextjs-with-typescript/pages/MuxPlayer.tsx

```diff
@@ -209,20 +209,52 @@ function MuxPlayerPage() {
           />
         </div>
         <div>
-          <label htmlFor="controlslist-control">Controlslist </label>
+          <label htmlFor="controlsvars-control">Hide controls CSS vars </label>
           <select
-            id="controlslist-control"
+            id="controlsvars-control"
             multiple
-            onChange={(event) => setControlslist(
-              Array.from(event.target.selectedOptions)
-                .map(({ value }) => value).join(' ')
+            onChange={(event) => setSelectedCssVars(
+              Object.fromEntries(Array.from(event.target.selectedOptions)
```

Comment:
idea(non-blocking): cool. Fyi `useState` supports functional updates in case you wanted to simplify this: https://reactjs.org/docs/hooks-reference.html#functional-updates